satisfactory/egg-pterodactyl-satisfactory.json

```diff
@@ -90,6 +120,16 @@
             "rules": "required|numeric|min:1",
             "field_type": "text"
         },
+        {
+            "name": "[Advanced] Network Quality",
```

Comment:
is part of the new Server GUI

Reply:
Yeah I removed this, along with the other vars.